tox.ini

```diff
@@ -43,6 +43,7 @@ deps =
     Sphinx
 
 [pep8]
+ignore = E123,E266
```

Comment:
Just saw the "In the default configuration, the checks E123, E133, E226, E241 and E242 ..." part in the [`pep8` intro][1]. But, what does `E123` have to do with this PR?

As for `E266`, we can leave it enabled and just not use `##` for the license.

Similarly simple hack: `# BEGIN: LICENSE` and `# END: LICENSE` hacks.

[1]: https://pep8.readthedocs.org/en/1.4.6/intro.html